docs/src/user-guide/configuring/rules.md

```diff
@@ -20,6 +20,8 @@ To change a rule's severity, set the rule ID equal to one of these values:
 * `"warn"` or `1` - turn the rule on as a warning (doesn't affect exit code)
 * `"error"` or `2` - turn the rule on as an error (exit code is 1 when triggered)
 
+The `"warn"` severity is meant to be used as a temporary measure. It can be used to enable a rule so that new reported issues can be addressed, without breaking the build due to existing ones. It is recommended to keep the number of these issues low so that new issues get more easily noticed and don't get ignored.
```

Comment:
I think we haven't decided yet whether this should be the only advice for using warnings: https://github.com/eslint/eslint/issues/16696

Reply:
Ah I hadn't seen that issue. Thanks.

Reply:
I feel we can add `usually` here to address that this is not with every case but most of the cases.

``` suggestion
The `"warn"` severity is usually meant to be used as a temporary measure. It can be used to enable a rule so that newly reported issues can be addressed, without breaking the build due to existing ones. It is recommended to keep the number of these issues low so that new issues get more easily noticed and don't get ignored.
```

Reply:
I'm fine with that, but then I think we should also indicate the other potential uses (just like @bmish said in a comment below)

Reply:
Let's not make this change piece-by-piece. I'll take a stab at writing up what I think is the best advice for using "warn".